tests/testthat/test-sas-to-parquet.R

```diff
@@ -1,44 +1,43 @@
-library(dplyr)
-
+# Prepare paths to temporary files.
 temp_sas_file <- fs::path_temp("test.sas7bdat")
 temp_sas_file_year <- fs::path_temp("test2019.sas7bdat")
 temp_parquet_file <- fs::path_temp("test.parquet")
 temp_parquet_partition <- fs::path_temp("test", "year=2019", "part-0.parquet")
 
+co2_df <- CO2 |>
+  dplyr::mutate(across(where(is.factor), as.character)) |>

```

Comment:
Needs namespacing for across and where too. Not sure why this didn't cause an error though :thinking: 

```suggestion
  dplyr::mutate(dplyr::across(tidyselect::where(is.factor), as.character)) |>
```